examples/yolol/operations.yolol

```diff
@@ -15,6 +15,6 @@ goto counter-8 :testgotovar=0
 :testneq=1!=42 and not (1!=1)
 :testgt=2>1 and not (1>2) and 5> -5
 :testgte=2>=1 and not (1>=2) and 2>=2
-:testlt=1<2 and not (2<1) and -5<5
```

Comment:
While that does now properly test < for strings, you removed the test-case for numbers.
I would suggest adding seperate testcases for <,>,>=,<= for strings and keep the tests for numbers unchanged. As this file already has 20 lines you would need to create a new file operations4.yolol and a operations4_test.yaml for it.